v2/cmd/nuclei/main.go

```diff
@@ -205,6 +205,7 @@ on extensive configurability, massive extensibility and ease of use.`)
 		flagSet.StringVar(&options.CustomConfigDir, "config-directory", "", "override the default config path ($home/.config)"),
 		flagSet.IntVarP(&options.ResponseReadSize, "response-size-read", "rsr", 10*1024*1024, "max response size to read in bytes"),
 		flagSet.IntVarP(&options.ResponseSaveSize, "response-size-save", "rss", 1*1024*1024, "max response size to read in bytes"),
+		flagSet.StringVarP(&options.Code, "code", "co", "", "code template trust store path (the file must contain each template name - one per line)"),
```

Comment:
this might be complex , discussion at https://github.com/projectdiscovery/nuclei/issues/2964 
@ehsandeep  had some ideas regarding this

Reply:
This should be indirectly mitigated with https://github.com/projectdiscovery/team-backlogs/issues/238